events/events-team/content/new-contributor-workshops/live-pr-demo.md

```diff
@@ -6,7 +6,7 @@ Plan for about 30 minutes
 
 ### Task Overview
 
-Demonstrate opening a pull request against k/k. There are some docs cleanup good-first-issues, but feel free to show off something you've been working on for a while.
+Demonstrate opening a pull request against the Kubernetes/Kubernetes(k/k) repository. There are some docs cleanup good-first-issues, but feel free to show off something you've been working on for a while.
```

Comment:
Could add a link here to git.k8s.io/kubernetes aka github.com/kubernetes/kubernetes.  "k/k" is a frequently used piece of jargon likely unclear to a new contributor who might be reading this outline.